ci/test-vmc.sh

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+WORK_HOME="/var/lib/jenkins"
+CLUSTER=""
+KUBECONFIG_PATH=$WORK_HOME/.kube/config
+MODE="report"
+RUN_GARBAGE_COLLECTION=false
+RUN_SETUP_ONLY=false
+RUN_CLEANUP_ONLY=false
+TESTCASE=""
+SECRET_EXIST=false
+TEST_FAILURE=false
+CLUSTER_READY=false
+
+_usage="Usage: $0 [--cluster-name <VMCClusterNameToUse>] [--kubeconfig <KubeconfigSavePath>] [--work-home <HomePath>]
+                  [--log-mode <SonobuoyResultLogLevel>] [--testcase <e2e|conformance|whole-conformance|networkpolicy>]
+                  [--garbage-collection] [--setup-only] [--cleanup-only]
+
+Setup a VMC cluster to run K8s e2e community tests (E2e, Conformance, whole Conformance & Network Policy).
+
+        --cluster-name           The cluster name to be used for the generated VMC cluster. Must be specified if not run in Jenkins environment.
+        --kubeconfig             Path to save kubeconfig of generated VMC cluster.
+        --work-home              Home path for Go, vSphere information and antrea_logs during cluster setup. Default is $WORK_HOME.
+        --log-mode               Use the flag to set either 'report', 'detail', or 'dump' level data for sonobouy results.
+        --testcase               The testcase to run: e2e, conformance, whole-conformance or networkpolicy.
+        --garbage-collection     Do garbage collection to clean up some useless testbeds
+        --setup-only             Only perform setting up the cluster and run test.
+        --cleanup-only           Only perform cleaning up the cluster."
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+function print_help {
+    echoerr "Try '$0 --help' for more information."
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --cluster-name)
+    CLUSTER="$2"
+    shift 2
+    ;;
+    --kubeconfig)
+    KUBECONFIG_PATH="$2"
+    shift 2
+    ;;
+    --work-home)
+    WORK_HOME="$2"
+    KUBECONFIG_PATH=$WORK_HOME/.kube/config
```

Comment:
I wouldn't recommend doing this here. If the user provides both `--kubeconfig` and `--work-home`, the behavior may be unexpected based on the order of these options.

Reply:
Indeed. I will remove this line here. And after this block of code, new code will check if `WORK_HOME` is changed and `KUBECONFIG_PATH` is not, then `KUBECONFIG_PATH` should be `$WORK_HOME/.kube/config`